examples/load/get.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
```

Comment:
N.B.: The example uses this shell script to load files at runtime but the idea is to provide a set of [ptex](https://github.com/a-scie/ptex) binaries for all supported platforms that can be included in a scie instead. That will free the scie from having dependencies on the target system providing `bash`, `curl` and `jq`.